site/content/entry/citra-progress-report-2017-september.md

```diff
@@ -0,0 +1,360 @@
++++
+date = "2017-11-01T17:40:00-04:00"
+title = "Citra Progress Report - 2017 September"
+tags = [ "progress-report" ]
+author = "anodium"
+forum = 0
++++
+
+Fall arrives once more, and like I mentioned in [last month's progress report](/entry/citra-progress-report-2017-august),
+I am extremely excited for what's in store. In fact, many of the really big goodies
+I've decided to seperate to their own articles, which should be coming up in the
+next few weeks.
+
+There's also been many changes this month that improve the speed of emulation across
+the board, on top of the usual improvements in accuracy and features. And because
+of that, I've dubbed this month #Speedtember. Let's dive right in.
+
+## [Switchable Page Tables](https://github.com/citra-emu/citra/pull/2952) by [MerryMage](https://github.com/MerryMage)
+
+Citra has a component called [dynarmic](https://github.com/MerryMage/dynarmic),
```

Comment:
This isn't quite what this PR is doing, Citra already used fastmem in dynarmic, but it was temporarily disabled when #2842 was merged because the JIT didn't support changing the page table at runtime like the rest of citra. What this PR does is it restores that fastmem functionality with the new switchable page tables code.

Reply:
Already discussed with @MerryMage that citra-emu/citra#2952 reimplements work that had already been done, but decided it would be best to leave it like this for simplicity's sake.